src/components/dialog/dialog.scss

```diff
@@ -37,8 +37,8 @@ $dialog-sizes: (
   box-shadow: 0 10px 20px rgba($grey-shadow, 0.2);
   position: fixed;
   top: 50%;
-  z-index: $z-dialog;
-
+  height: calc(95%);
```

Comment:
this breaks the height of the dialogs

![screen shot 2018-02-01 at 11 41 33](https://user-images.githubusercontent.com/1668080/35676753-eae2891e-0744-11e8-9050-c9925690b42e.png)